src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -567,9 +567,10 @@ actionFrontControllerSetVariables
     ```php
       <?php
       array(
-        'templateVars' => &(array)
+        'templateVars' => &(array) template variables
       );
     ```
+    There was no parameter before 1.7.7
```

Comment:
I wonder if this is really necessary?
You have just above :
`Parameters since {{< minver v="1.7.7" >}}`

Reply:
I thought it was the reference (&) stuff that was new.
for the // : others samples are not like that

Reply:
I think you can remove this line @jf-viguier 

If there's an indicator that this hook is from 1.7.7 then why we need this comment?